microservice-transaction-sample/rpc/build.gradle

```diff
@@ -20,16 +20,40 @@ protobuf {
     generateProtoTasks {
         all()*.plugins { grpc {} }
     }
-    generatedFilesBaseDir = "$projectDir/src"
 }
 
-archivesBaseName = "sample-rpc"
+base {
+    archivesName = "sample-rpc"
+}
+
+// Task copyGeneratedProtoToSrc copies the generated .java files into src directory
+task copyGeneratedProtoToSrc(type: Copy) {
+    description 'Copies generated Protocol Buffer classes to src/main/java/sample/rpc'
+    dependsOn generateProto
+    from "$buildDir/generated/source/proto/main/java/sample/rpc"
+    into "$projectDir/src/main/java/sample/rpc"
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+}
```

Comment:
Since `generatedFilesBaseDir` is deprecated, we need to implement a similar functionality using copy.